R/plots-qc.R

```diff
@@ -113,18 +120,32 @@ qc_variance_hist <- function(gimap_dataset, wide_ar = 0.75){
 #' @examples \dontrun{
 #' gimap_dataset <- get_example_data("gimap")
 #' qc_constructs_countzero_bar(gimap_dataset)
+#' 
+#' #or if you want to select a specific column(s) for looking at where/which samples zero counts are present for
+#' qc_constructs_countzero_bar(gimap_dataset, filter_zerocount_target_col = c(3:5))
```

Comment:
You shouldn't need to have the `c()` here?

Reply:
Yeah I think you're right. I just hadn't gone through and removed them from all the PRs following the code review last week.

Reply:
pushed a change that simplifies the ones I found